test.js

```diff
@@ -4,17 +4,18 @@ var sparkly = require('./');
 
 test('creates graph', function (t) {
 	t.assert(sparkly([1, 5, 22, 13, 5]) === '▁▂█▅▂');
-	t.assert(sparkly([0, 30, 55, 80, 33, 150]) === '▁▂▃▄▂█');
-	t.assert(sparkly([5.5, 20]) === '▁█');
+	t.assert(sparkly([0, 30, 55, 80, 33, 150]) === '▁▂▃▅▂█');
+	t.assert(sparkly([5.5, 20]) === '▃█');
```

Comment:
20 is almost 5.5, 3½ times!
The column index for representing 5.5 is somewhere between index 2 and 3. 
If the selected index is 2, that's going to be 3 x 5.5 = 16.5 which is smaller than 20.
if the selected index is 3, that's going to be 4 x 5.5 = 22 which greater that 20.
I think what happened here is that index 3 was selected because 22 is closer to 20 which makes 20 is 5.5 times 4.
Since the indexes are representing whole numbers, the columns representation will not be very accurate.